crypto/s2n_composite_cipher_aes_sha.c

```diff
@@ -167,17 +167,21 @@ static int s2n_composite_cipher_aes_sha_encrypt(struct s2n_session_key *key, str
     POSIX_ENSURE_EQ(out->size, in->size);
 
     POSIX_GUARD_OSSL(EVP_EncryptInit_ex(key->evp_cipher_ctx, NULL, NULL, NULL, iv->data), S2N_ERR_KEY_INIT);
-    POSIX_GUARD_OSSL(EVP_Cipher(key->evp_cipher_ctx, out->data, in->data, in->size), S2N_ERR_ENCRYPT);
+
+    int len = out->size;
```

Comment:
```suggestion
    int len = 0;
```
It's an output parameter. Not sure why other parts of the codebase are initializing it to the size of the blob.